### ISSUE TITLE
Variable name minification 

### ISSUE THREAD

**Issue by mohsen1** [parcel-bundler/parcel-css#27]
I'm not sure if this is inline with project goals but I've noticed modern CSS has a lot of very long variable names. If variables are declared in the same file it is somewhat safe to rename the variable names to something shorter. 

It's worth noting that shorter variable names are often not that useful for assets that are gzipped because Gzip does this optimization.

**Comment by devongovett** [comment]
Yeah it's a good idea, but would need to be opt-in. Variables can be accessed from other CSS files, or from JavaScript.

I was thinking about doing this for CSS modules though. Variables could be hashed to avoid conflicts with other files, and then minified. The only problem is we'd need a syntax to be able to opt out of this and access variables from other files or globals. For example, like the `composes: classname from 'other.css'` syntax.
